doc/requirements.tex

```diff
@@ -10,11 +10,12 @@
     \item Annotation blocks must be located in a way that it can easily be skipped by model-unaware clients.
     \item The vocabulary in the annotation name-space must not overlap with the VOTable elements (names or attributes)    
     \item The annotation syntax must be able to inform the client about the status of the annotation process.
+          It needs a place where we can stick error messages in case the annotation went wrong.
```

Comment:
"stick" is nice and might even be my language, but perhaps it's a bit too easy-going for this kind of text.  Perhaps "The annotation must provide an element for communicating success or failure of the annotation and possible diagnostics" (where I'd say this whole thing is somewhat too much inspired by the idea that DM annotation is a separate process, which I think it's not in general; but I'd not mind that very much).

Reply:
rephrased according to your suggestion. 
note: if the annotation is not a separate process but somehow hardcoded, just let the status as OK